bin/pycbc_inspiral

```diff
@@ -197,6 +196,12 @@ parser.add_argument("--checkpoint-exit-maxtime", type=int,
                          " time is exceeded. Default is no checkpointing.")
 parser.add_argument("--checkpoint-exit-code", type=int, default=77,
                     help="Exit code returned if exiting after a checkpoint")
+parser.add_argument("--multiprocessing", type=int,
+                    help="Parallelize over multiple processes, note this is"
+                         "separate from threading using the proc. scheme")
+parser.add_argument("--template-groupby", type=int, default=1,
+                    help="Number of template to group together for parallel"
```

Comment:
templates, typo

Reply:
Option removed in favor of using the finalize events template rate option